wordle-web/pages/index.vue

```diff
@@ -0,0 +1,13 @@
+<template>
+  <v-card>
+    <v-card-item>
+      <v-card-title class="text-center" > Welcome to this beautiful website! </v-card-title>
+      <v-card-subtitle>
+        Click the game button... <strong class="text-red-accent-4"> if you dare...</strong>
+      </v-card-subtitle>
+    </v-card-item>
+    <v-card-text>
+      <NuxtLink to="/game"> Game </NuxtLink>

```

Comment:
I love the early 2000s vibe of clickable text instead of a styled button. Thanks for the nostalgia!